app/ProjectAanvraag/Listeners/CreateWidget.php

```diff
@@ -139,4 +140,24 @@ private function handle(UuidInterface $integrationId): void
             )
         );
     }
+
+    public function failed(IntegrationCreated|ContactCreated|ConsumerCreated $event, Throwable $throwable): void
+    {
+        $entity = match (get_class($event)) {
+            IntegrationCreated::class => 'integration',
+            ContactCreated::class => 'contact',
+            ConsumerCreated::class => 'consumer',
+        };
+
+        $id = match (get_class($event)) {
+            IntegrationCreated::class => $event->integrationId->toString(),
+            ContactCreated::class => $event->id->toString(),
+            ConsumerCreated::class => $event->id->toString(),
+        };
+
+        $this->logger->error("Failed to create $entity", [
```

Comment:
What always fails is the widget creation. The trigger is one of `IntegrationCreated|ContactCreated|ConsumerCreated`

Reply:
[Change log to say widget failed](https://github.com/cultuurnet/publiq-platform/pull/906/commits/8eb57ec67ffb24a73ee0d3a204518030e1b9b1e8)